src/content/en/2021/caching.md

```diff
@@ -17,4 +19,205 @@ featured_stat_3: TODO
 featured_stat_label_3: TODO
 ---
 
-## TODO
+## Introduction
+
+Caching is a technique that enables the reuse of previously downloaded content. Over the past decade...
+
```

Comment:
The three main pillars of caching web content are: 
- Cache as much as you can
- Cache for as long as you can
- Cache as close to end users as you can